### DISCUSSION TITLE
Crash on opening

### DISCUSSION THREAD

**Discussion by spiffo-smith** [inteyes/ReMS2000#1] [General]
Hi, 
![Callback Error](https://user-images.githubusercontent.com/83515811/163432097-e59bb6a4-656d-4c4d-9002-7978530f3310.png)
![Callback Error 2](https://user-images.githubusercontent.com/83515811/163432373-adb535f3-aa44-48c8-8c72-f0eddc7d5897.png)
![Timer Callback Error](https://user-images.githubusercontent.com/83515811/163432564-87ec2f53-2b84-4859-b569-a14c042a1398.png)

Getting these errors when launching your Panel!

Looking at the LUA, looks like your methods: startupSequence() and loadStateOperations() have crashed!

However the line 6 getLocalSettingsFile error seems to be whats called on line 6 of restoreGlobalSettings() where you call the function getLocalSettingsFile() and looking in there you seem to be trying to load a file called **config.json** which I don't have!

Any ideas?

Thanks, Graham.

**Comment by inteyes** [comment]
Hello!

Thank you for the information.
Looks like I forgot to add a check for the existence of the config.json file before trying to access it. I already fixed this and I will publish fixed instances this weekend (need some free time to prepare and export them).

As a quick fix you can open and close the "Settings" page of the panel, and then reload the panel. The required file will be created.

**Reply by spiffo-smith** [reply]
OK, that's great, I was hoping it would be that simple!
